kurtosis-module/impl/module_io/params.go

```diff
@@ -89,6 +89,17 @@ type ExecuteParams struct {
 	// If set, the module will block until a finalized epoch has occurred
 	WaitForFinalization bool	`json:"waitForFinalization"`
```

Comment:
This is now only used if the `WaitForVerifications` is set to `false`, so would be good to update this comment and the README so users know what's up